tests/trace_service.py

```diff
@@ -0,0 +1,86 @@
+import concurrent
+import grpc
+from opentelemetry.proto.collector.trace.v1 import trace_service_pb2
+from opentelemetry.proto.collector.trace.v1 import trace_service_pb2_grpc
+import pytest
+import subprocess
+import time
+
+
+class TraceService(trace_service_pb2_grpc.TraceServiceServicer):
+    batches = []
+
+    def Export(self, request, context):
+        self.batches.append(request.resource_spans)
+        return trace_service_pb2.ExportTracePartialSuccess()
+
+    def get_span(self):
+        for _ in range(10):
+            if len(self.batches):
+                break
+            time.sleep(0.01)
```

Comment:
The rate here should match Nginx's 1ms. We can increase attempts count, though.

Reply:
OK, aligned with nginx.